src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+abstract class Figure implements AreaCalculator, DrawInfo {
+    private String color;
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+
+        FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a `new FigureSupplier()` inside the loop. Although you've done this correctly, the `main` method should not be in the `Figure` class.